grammar.js

```diff
@@ -251,25 +251,25 @@ module.exports = grammar({
       field("condition", $._expression),
       ")",
       repeat($._newline),
-      field("body", optional($._expression))
+      optional(field("body", $._expression))
     )),
 
     repeat: $ => prec.right(seq(
       "repeat",
       repeat($._newline),
-      field("body", optional($._expression))
+      optional(field("body", $._expression))
     )),
 
     // Blocks.
     "{": $ => prec.right(seq(
       "{",
-      field("body", repeat(choice($._expression, $._semicolon, $._newline))),
+      repeat(field("body", choice($._expression, $._semicolon, $._newline))),
```

Comment:
@kevinushey I was never able to figure out a clean way to get everything into a single `body` that had the statements as children - so instead I have left the current behavior as is (multiple `body` nodes, 1 per statement) and switched this around to try and clarify the meaning of what is actually happening. It also looks like we had some other code that expected this to work this way already.